src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -99,12 +100,12 @@ static PSVersionInfo()
                 rawGitCommitId = mainVersion;
             }
 
-            s_psV6Version = new SemanticVersion(mainVersion);
+            s_psV7Version = new SemanticVersion(mainVersion);
 
-            s_psVersionTable[PSVersionInfo.PSVersionName] = s_psV6Version;
+            s_psVersionTable[PSVersionInfo.PSVersionName] = s_psV7Version;
             s_psVersionTable[PSVersionInfo.PSEditionName] = PSEditionValue;
             s_psVersionTable[PSGitCommitIdName] = rawGitCommitId;
-            s_psVersionTable[PSCompatibleVersionsName] = new Version[] { s_psV1Version, s_psV2Version, s_psV3Version, s_psV4Version, s_psV5Version, s_psV51Version, s_psV6Version };
+            s_psVersionTable[PSCompatibleVersionsName] = new Version[] { s_psV1Version, s_psV2Version, s_psV3Version, s_psV4Version, s_psV5Version, s_psV51Version, s_psV6Version, s_psV7Version };
```

Comment:
Should we remove s_psV6Version?

Reply:
PS7 is compatible with PS6, so it needs to be there